src/pages/EditorPage.js

```diff
@@ -314,13 +446,12 @@ export default function EditorPage(props) {
     (path, code) => {
       try {
         const formattedCode = prettier.format(code, {
-          parser: "babel",
+          parser: 'babel',
           plugins: [parserBabel],
         });
         updateCode(path, formattedCode);
-      } catch (e) {
-        console.log(e);
-      }
+        checkForkDetails(path);
+      } catch (e) {}
```

Comment:
why remove the logging of these errors?

Reply:
This is meant to be a console.error and I removed all console logs as part of cleaning up the code before committing.